### PR TITLE
Add reset button to flow page

### PR DESCRIPTION
**Pull request by CathrinTruchan**
I added a reset button to the flow page. It is only displayed after the user added poses to the flow. When the user clicks on it, the flow the flow is emptied.

### CODE REVIEW

File: pages/flow/index.js [hunk 1]
```diff
@@ -35,7 +36,22 @@ export default function Flow() {
...
+          style={{ margin: "-15rem auto 5rem auto" }}
+        >
+          + Add Asanas
+        </MainButton>
+      )}
+      {!open && flowAsanaIds.length > 0 && (
+        <MainButton
+          type="secondary"
+          onClick={() => setFlowAsanaIds([])}
+          style={{ margin: "-3rem auto 5rem auto" }}
```
Comment (sebsch1337) [inline pages/flow/index.js:51]: You should use properties for styled components instead of inline styling :)

**Review by sebsch1337 — CHANGES_REQUESTED** [review]
Looks good - just a small and a bigger change :)

File: pages/flow/index.js [hunk 2]
```diff
@@ -35,7 +36,22 @@ export default function Flow() {
...
+        >
+          X Rest flow
```
[the remark this answers is not in the document]
Comment (CathrinTruchan) [reply pages/flow/index.js:53]: Thanks for the hint :)

**Review by sebsch1337 — APPROVED** [review]
LGTM

**Pull request merged**